open_gateway/app.py

```diff
@@ -678,12 +696,37 @@ def main():
                 app.config["MODEL_JSON"] = json.load(open(arg))
             else:
                 print("Model json file was not found!")
+        elif opt in ("-i", "--classmap_images_json_path"):
+            print(f"{C_CLR_OKBLUE}Start saving classmap images:")
+            if os.path.exists(arg):
+
+                images_read = json.load(open(arg))
+                app.config["CLASS_MAP_IMAGES"] = []
+                dir_to_save  = os.path.join(app.static_folder, CLASS_MAP_IMG_FLD_NAME)
+
+                image_manager = ImageManager(dir_to_save=dir_to_save)
+
+                for class_name, img_path in images_read.items():
+                    try:
+                        # save image into the static folder
+                        new_img_name = image_manager.resave_img(img_path, img_name="_".join(class_name.lower().split()))
+                    except ImageManager.errors as e:
+                        print(f"{C_CLR_ERROR}{e}")
+                        sys.exit()
+                    else:
+                        print(f"{C_CLR_OKBLUE} Saved image for class {class_name}")
+                    app.config["CLASS_MAP_IMAGES"].append({ "name": class_name, "img": new_img_name })
+            else:
+                print(f"{C_CLR_ERROR} Classmap images json file was not found!")
+                sys.exit()
```

Comment:
put a sleep timer before exiting. otherwise they won't see the message before the window closes